src/pages/home.jsx

```diff
@@ -15,8 +16,7 @@ export const Home = props => {
   const [data, setData] = useState(null);
 
   useEffect(() => {
-    API({
-      url: '/posts',
+    PICKLY.get('/posts', {
       headers: {
         authorization: `bearer ${token}`
       }
```

Comment:
- don't pass token here, it caused an error, and it's already passed at the **PICKLY** function.
- please pull the last develop branch changes to your branch.